about.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CC Global Summit 2049</title>
+    <link rel="stylesheet" href="stylesAbout.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons"/>
+    <link href="media/favicon.ico" title="Icon" type="image/x-icon" rel="icon">
+</head>
+<body>
+    <header>
+        <section class="flexRow" id="top">
+            <img src="media/buttons/facebook.svg" alt="facebook handle">
+            <img src="media/buttons/twitter.svg" alt="twitter handle">
+            <p>English</p>
+            <p>My Page</p>
+            <p>Logout</p>
+        </section>
+        <section class="flexRow" id="nav">
+            <a href="index.html" id="logo">
+                <img src="media/logo.png" alt="cc global logo">
+            </a>
+            <a href="about.html"><p>About</p></a>
+            <a href="#"><p>Program</p></a>
+            <a href="#"><p>Join</p></a>
+            <a href="#"><p>Sponsor</p></a>
+            <a href="#"><p>News</p></a>
+            <button>CC Campaign</button>
+        </section>
+    </header>
+    <section id="hero">
+        <img src="media/buttons/menu.svg" alt="menu" id="mobileMenuButton" onclick="mobileMenu()">
+        <div class="head">
+            <a href="index.html">
+                <p>"Hello! Sharing world"</p>
+                <h1>COOKING CHANNEL GLOBAL SUMMIT 2049</h1>
+            </a>
+        </div>
+        <article>
+            <p>In particular, this year's CC Summit is waiting for the participation of various organizations
+                 and individual participants who want to participate in activities to spread the value of sharing, 
+                 free culture, and open knowledge with CC . Free protein supporters, Wallace employees, 
+                 open knowledge foundation activists, art galleries, libraries, museums, archives, governments, 
+                 foundations, lawyers, protein farmers, etc. various people active in the field of protein cooking 
+                 and protein farming gather in one place to share. We hope that it will be an opportunity to share 
+                 our thoughts on creating an ecosystem.</p>
```

Comment:
* I suggest you apply the proper font family where necessary as the one you are using (`sans-serif`) is different from the original design 
![image](https://user-images.githubusercontent.com/52497246/128958573-fecf1472-6322-471b-bf81-7e1d50e58743.png)

Reply:
The proper font family costs €265, so I hope you understand it's not possible to use the proper font family. I used something that looks pretty similar.
![cocofont price](https://user-images.githubusercontent.com/81324033/129070183-dbcf8388-381e-4528-8cf6-6705985b508a.jpg)

Reply:
But if you don't mind sharing with me a way to get that font with all of its characters and styles for free, I'll be happy to modify my project.

Reply:
- [ ] [Here](https://www.dafont.com/es/cocogoose.font) you can find the free font.
- A simple google trying to find the font could help you on this point. I really invite you to use google in a better way. This is not only for fonts but also for a web developer, using google in a better way will help you to solve issues in less time. Google is an important tool as a developer
![image](https://user-images.githubusercontent.com/20563252/129241343-e25513b8-12ac-4ed5-b497-4c9e9c06237a.png)

Reply:
Following the link you shared with me to get the free font takes me to dafont.com; I had looked at that website before, the problem is that you can't get the full version of the font there, as you can see in the attached screenshot, you don't get numbers in a regular font style with this download, I find those to be crucial for this project, that's why I went with a similar font instead.

![cocogoose in dafont](https://user-images.githubusercontent.com/81324033/129246902-b04af751-8308-499b-983f-8f80d77018f1.jpg)

Furthermore, following the second link you shared with me takes me to the licencing in zetafonts(the creators of the cocogoose font). The problem is the same as stated before; the personal use licence is the one that costs €265, as you can see in the screenshot below. These are the two reasons I was unable to use the original font for this project. 

![ksnip_20210812-130503](https://user-images.githubusercontent.com/81324033/129247416-31f25ff0-9952-498f-af60-18964683d2a5.png)